pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/GjenlevendeInfoEtter1970.kt

```diff
@@ -73,38 +73,38 @@ object GjenlevendeInfoEtter1970 : AutobrevTemplate<Unit> {
+                    English to "The Norwegian Labour and Welfare Administration (NAV) offers various services and support schemes for those who need help finding a job. You can find job search tips and a hyperlink to open jobs online at $ARBEID_URL.",
                 )
             }
             paragraph {
                 text(
                     Bokmal to "Vi kan snakke med deg om ulike muligheter i arbeidsmarkedet, utdanning eller andre tiltak.",
                     Nynorsk to "Vi kan snakke med deg om ulike moglegheiter i arbeidsmarknaden, utdanning eller andre tiltak.",
-                    English to "We can give advice about the opportunities available to you, whether it be employment, education or other initiatives.",
+                    English to "We are available for a conversation about the various opportunities available to you in the labour market, education, or other initiatives.",
                 )
             }
 
             paragraph {
                 text(
                     Bokmal to "Hvis du trenger utdanning som blir godkjent av NAV, eller du får arbeidstrening via NAV, kan du søke om skolepenger og tilleggsstønad. Du finner mer informasjon om disse stønadene på $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
                     Nynorsk to "Viss du treng utdanning som blir godkjent av NAV, eller viss du får arbeidstrening via NAV, kan du søkje om skolepengar eller tilleggsstønad. Du finn meir informasjon om desse stønadene på $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
-                    English to "If you need to attain additional education and NAV approves your education plan, or if you are participating in employment training from NAV, you can apply to have tuition fees or other supplementary benefits covered by NAV. You can find more information about these benefits at $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
+                    English to "If you are undertaking an education that NAV means is necessary for you to get a job and support yourself, you may get additional benefits. The same applies if you are registered as a jobseeker with NAV. You can find more information about these benefits at $GJENLEVENDE_TILLEGGSSTOENADER_URL.",
                 )
             }
 
             paragraph {
                 text(
                     Bokmal to "Hvis du fortsatt tar utdanning eller har arbeidstrening fra 1. januar 2027, kan du søke om å få pensjonen din forlenget med opptil to år.",
                     Nynorsk to "Om du framleis tar utdanning eller har arbeidstrening frå 1. januar 2027, kan du søkje om å få pensjonen din forlenga med inntil to år.",
-                    English to "If you are still studying or have work training from 1 January 2027, you can apply to have your pension extended by up to two years.",
+                    English to "If you are still undertaking an education or require help to find a job as of 1 January 2027, you can apply to have your pension extended by up to two years.",
```

Comment:
You might "require help to find a job" without actually undergoing "arbeidstrening". It's a NAV program that you either are undergoing or not.